src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -11,7 +11,7 @@
 import java.util.ArrayList;
 import java.util.List;
```

Comment:
import java.util.ArrayList and import javax.validation.constraints.NotNull not needed anymore...